src/canister/individual_user_template/src/data_model/mod.rs

```diff
@@ -163,6 +175,23 @@ impl Default for CanisterData {
             ml_feed_cache: Vec::new(),
             cdao_canisters: Vec::new(),
             token_roots: _default_token_list(),
+            proof_of_participation: None,
+            pubkey_cache: _default_pubkey_cache(),
+            airdrop: AirdropData::default(),
         }
     }
 }
+
+impl PoPStore<Memory> for CanisterData {
```

Comment:
Should we use better descriptive names?

Reply:
PoPStore came from Proof Of Pariticipation Store, but let me know if you have better ideas